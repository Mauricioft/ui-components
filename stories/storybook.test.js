import initStoryshots from '@storybook/addon-storyshots'
initStoryshots()

// import React from 'react'
// import renderer from 'react-test-renderer'
// import { Button } from '../atoms/Button'

// describe('Button component', () => {
// 	it('Should match the snapshot', () => {
// 		const component = renderer.create(<Button type='primary'>Primary</Button>)
// 		const tree = component.toJSON()
// 		expect(tree).toMatchSnapshot()
// 	})
// })
