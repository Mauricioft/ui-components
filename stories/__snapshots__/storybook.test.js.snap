// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Atoms/Button Primary 1`] = `
<button
  className="button type-primary is-block"
  onClick={[Function]}
>
  Primary
</button>
`;

exports[`Storyshots Atoms/Button Secondary 1`] = `
<button
  className="button type-secondary is-block"
  onClick={[Function]}
>
  Secondary
</button>
`;

exports[`Storyshots Atoms/Button Tertiary 1`] = `
<button
  className="button type-tertiary"
  onClick={[Function]}
>
  Tertiary
</button>
`;

exports[`Storyshots Atoms/Heading Extra Large 1`] = `
<h1
  className="heading color-default size-xl"
>
  Heading
</h1>
`;

exports[`Storyshots Atoms/Heading Extra Small 1`] = `
<h1
  className="heading color-default size-xs"
>
  Heading
</h1>
`;

exports[`Storyshots Atoms/Heading Large 1`] = `
<h1
  className="heading color-default size-lg"
>
  Heading
</h1>
`;

exports[`Storyshots Atoms/Heading Medium 1`] = `
<h1
  className="heading color-default size-md"
>
  Heading
</h1>
`;

exports[`Storyshots Atoms/Heading Primary 1`] = `
<h1
  className="heading color-primary size-md"
>
  Heading
</h1>
`;

exports[`Storyshots Atoms/Heading Small 1`] = `
<h1
  className="heading color-default size-sm"
>
  Heading
</h1>
`;

exports[`Storyshots Atoms/Heading Two Extra Large 1`] = `
<h1
  className="heading color-default size-2xl"
>
  Heading
</h1>
`;

exports[`Storyshots Atoms/Icon Angle Down 1`] = `
<div
  className="icon"
  style={
    Object {
      "height": 15,
      "width": 15,
    }
  }
>
  <picture
    className="picture"
  >
    <img
      src="icons/angle-down.svg"
      style={
        Object {
          "maxWidth": "15px",
        }
      }
    />
  </picture>
</div>
`;

exports[`Storyshots Atoms/Icon Arrow Right 1`] = `
<div
  className="icon"
  style={
    Object {
      "height": 15,
      "width": 15,
    }
  }
>
  <picture
    className="picture"
  >
    <img
      src="icons/arrow-right.svg"
      style={
        Object {
          "maxWidth": "15px",
        }
      }
    />
  </picture>
</div>
`;

exports[`Storyshots Atoms/Icon Has Background 1`] = `
<div
  className="icon has-background"
  style={
    Object {
      "height": 26,
      "width": 26,
    }
  }
>
  <picture
    className="picture"
  >
    <img
      src="icons/angle-down.svg"
      style={
        Object {
          "maxWidth": "26px",
        }
      }
    />
  </picture>
</div>
`;

exports[`Storyshots Atoms/Input Primary 1`] = `
<input
  className="input"
  onChange={[Function]}
  placeholder="Nombres"
  type="text"
  value=""
/>
`;

exports[`Storyshots Atoms/Picture Default 1`] = `
<picture
  className="picture"
>
  <img
    src="https://placeimg.com/240/480/any"
    style={
      Object {
        "maxWidth": "240px",
      }
    }
  />
</picture>
`;

exports[`Storyshots Molecules/ButtonIcon Secondary 1`] = `
<button
  className="button type-secondary is-block"
  onClick={[Function]}
>
  Secondary
  Â 
  <div
    className="icon"
    style={
      Object {
        "height": 15,
        "width": 15,
      }
    }
  >
    <picture
      className="picture"
    >
      <img
        src="icons/arrow-right.svg"
        style={
          Object {
            "maxWidth": "15px",
          }
        }
      />
    </picture>
  </div>
</button>
`;

exports[`Storyshots Molecules/Dropdown Default 1`] = `
<div
  className="dropdown"
>
  <div
    className="icon has-background"
    style={
      Object {
        "height": 26,
        "width": 26,
      }
    }
  >
    <picture
      className="picture"
    >
      <img
        src="icons/angle-down.svg"
        style={
          Object {
            "maxWidth": "26px",
          }
        }
      />
    </picture>
  </div>
  <select
    className="dropdown-select"
    onChange={[Function]}
    value="9:00 AM"
  >
    <option
      value={800}
    >
      8:00 AM
    </option>
    <option
      value={900}
    >
      9:00 AM
    </option>
    <option
      value={1000}
    >
      10:00 AM
    </option>
  </select>
</div>
`;

exports[`Storyshots layout/Spacer Extra Small 1`] = `
<div
  className="spacer is-visible"
  style={
    Object {
      "display": "inline-block",
      "height": 5,
      "width": 5,
    }
  }
/>
`;

exports[`Storyshots layout/Spacer Horizontal 1`] = `
<div
  className="spacer is-visible"
  style={
    Object {
      "display": "block",
      "height": 20,
      "width": "100%",
    }
  }
/>
`;

exports[`Storyshots layout/Spacer Large 1`] = `
<div
  className="spacer is-visible"
  style={
    Object {
      "display": "inline-block",
      "height": 40,
      "width": 40,
    }
  }
/>
`;

exports[`Storyshots layout/Spacer Medium 1`] = `
<div
  className="spacer is-visible"
  style={
    Object {
      "display": "inline-block",
      "height": 20,
      "width": 20,
    }
  }
/>
`;

exports[`Storyshots layout/Spacer None 1`] = `
<div
  className="spacer is-visible"
  style={
    Object {
      "display": "inline-block",
      "height": 0,
      "width": 0,
    }
  }
/>
`;

exports[`Storyshots layout/Spacer Small 1`] = `
<div
  className="spacer is-visible"
  style={
    Object {
      "display": "inline-block",
      "height": 15,
      "width": 15,
    }
  }
/>
`;

exports[`Storyshots layout/Spacer Vertical 1`] = `
<div
  className="spacer is-visible"
  style={
    Object {
      "display": "block",
      "height": "100vh",
      "maxHeight": 200,
      "width": 20,
    }
  }
/>
`;
